### PR TITLE
[terra-date-time-picker] Fixed accessibility issue

### PR DESCRIPTION
**Pull request by adavijit**
### Summary

**What was changed:**
- Added a div tag with `role="group"` to group the `DatePicker` and `TimeInput` component
- Fixed wdio test

**Why it was changed:**
`DatePicker` and `TimeInput` component was not grouped together leading to accessibility issue

### Testing

This change was tested using:

- [x] WDIO
- [x] Jest
- [x] Visual testing (please attach a screenshot or recording)
- [ ] Other (please describe below)
- [ ] No tests are needed

### Reviews

In addition to engineering reviews, this PR needs:

- [ ] UX review
- [x] Accessibility review
- [ ] Functional review

### Additional Details

**This PR resolves:**

UXPLATFORM-10229 

---

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-date-time-picker/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Fixed
+  * Fixed the accessibility issue in `DateTimePicker` component.
```
Comment (sugan2416) [inline packages/terra-date-time-picker/CHANGELOG.md:6]:
```suggestion
  * Fixed the accessibility issue to allow announcement as a single group label.
```

**Review by sugan2416 — APPROVED** [review]

**Review by allanxavierm — APPROVED** [review]

**Comment by rbsree** [comment]
+1 for accessibility review for Terra Date Time Picker, as suggested the Date and Time input field are grouped.

File: packages/terra-date-time-picker/src/DateTimePicker.jsx [hunk 2]
```diff
@@ -789,60 +789,59 @@ class DateTimePicker extends React.Component {
...
-            isInvalid={isInvalid}
-            isInvalidMeridiem={isInvalidMeridiem}
-            required={required}
-            atMaxDate={atMaxDate}
-            atMinDate={atMinDate}
-            errorId={errorId}
-          />
-
-          {this.state.isAmbiguousTime && this.state.dateTime ? this.renderTimeClarification() : null}
+        <div role="group" aria-label={ariaLabel}>
```
Comment (sycombs) [inline packages/terra-date-time-picker/src/DateTimePicker.jsx:792]: Do we need to add another div? Can we add the role and aria-label to the existing `<div className={cx('date-facade')}>` div?

**Comment by sycombs** [comment]
Can we also add some Jest tests for the inclusion of the `role` and `aria-label` attributes?

**Pull request merged**